src/address_helpers.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _ADDRESS_HELPERS_H_
+#define _ADDRESS_HELPERS_H_
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+bool get_public_key(uint32_t account_number, uint32_t index, uint8_t *public_key_array);
+void get_address_hex_from_binary(uint8_t *public_key, char *address);
+void get_address_bech32_from_binary(uint8_t *public_ey, char *address);
```

Comment:
The `const` qualifier should be added to the `public_key` parameter of these two functions.
There is also a small typo (`public_ey` instead of `public_key`).

Reply:
done + fixed the typo